src/tests/unit/pizzas-dao.test.js

```diff
@@ -6,14 +6,17 @@ import proxyquire from 'proxyquire';
 import sinon from 'sinon';
 import sinonChai from 'sinon-chai';
 
-import { getPizzasData, getPizzaByIdData, postPizzaData } from './test-data';
+import { withConnectionStubGenerator } from './with-connection-stub';
+import {
+  getPizzasData, getPizzaByIdData, postPizzaData, updatePizzaByIdData,
+} from './test-data';
 
 const should = chai.should();
 chai.use(chaiAsPromised);
 chai.use(sinonChai);
 
 let checkIngredientsExistsStub;
-let connectionStub;
+let executeStub;
```

Comment:
this has a huge diff because `connectionStub` was renamed to `executeStub`; real changes are at the bottom of the file